aten/src/ATen/cuda/CUDAApplyUtils.cuh

```diff
@@ -247,7 +247,7 @@ template <typename Op,
 struct ApplyOp1<Op, scalar, IndexType, ADims, 0, Offset> {
 __device__ __forceinline__
 static void apply(detail::TensorInfo<scalar, IndexType> &a, const Op &op,
-                  int n, IndexType linearIndex, Offset offset) {
+                  int /*n*/, IndexType /*linearIndex*/, Offset offset) {
```

Comment:
I vaguely remember there were a flavor of Windows compiler which treated such declarations as invalid. What's wrong with marking them `C10_UNUSED`?

Reply:
@malfet - if I mark them `C10_UNUSED` and they become used in the future no compiler warning I'm aware of will indicate that `C10_UNUSED` is no longer true. So it seems like using it invites desynchronization between warning suppressions and the actual code at the cost of readability.